azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md

```diff
@@ -24,14 +24,14 @@ Retrieves the object(s) specified by the objectIds parameter
 
 ### Example 1
 ```
-PS C:\WINDOWS\system32> Get-AzureADObjectByObjectId -ObjectIds 2af3478a-27da-4837-a387-b22b3fb236a8, c4fdf87f-f68e-4859-8bcf-36579b66005e
+PS C:\WINDOWS\system32> Get-AzureADObjectByObjectId -ObjectIds aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb, c4fdf87f-f68e-4859-8bcf-36579b66005e
```

Comment:
Not all Object Id values have been updated in the command and in the output as well.

Reply:
Ok for now.